src/and-cli.test.ts

```diff
@@ -13,35 +15,37 @@ describe("and-cli", () => {
     // -----------------------------------------------------------------------------------------
 
     describe("commands", () => {
-        // Pull out the flattened list of command objects from 'commands' module
-        const commandObjects = Object.keys(Commands).map(
-            (key) => Commands[key]
+        // Pull out the flattened list of command definitions
+        const commandDefinitions = CommandDefinitionUtils.flatten(
+            CommandDefinitions
+        );
+        const commandNames = CommandDefinitionUtils.getNames(
```

Comment:
Thought: If the parameter is left undefined (unsupplied), could the default behavior be that the utils simply grabs 'CommandDefinitions' for you? While I realize utils are a set of functions, "encapsulation" in ways still applies. Being the most likely scenario when using CommandDefinitionUtils is to leverage the `modules/command-definitions` file. Having the utils package import that and leverage whereever possible will make the public API of the corresponding utils file much friendlier.